javascriptv3/example_code/bedrock-runtime/tests/anthropic_claude.integration.test.js

```diff
@@ -0,0 +1,105 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect, describe, it } from "vitest";
+import { FoundationModels } from "../config/foundation_models.js";
+import { expectToBeANonEmptyString } from "./test_tools.js";
+import {
+  invokeMessagesApi as invokeClaudeInstantMessagesApi,
+  invokeTextCompletionsApi as invokeClaudeInstantTextCompletionsApi,
+} from "../models/anthropic_claude/claude_instant_1.js";
+import {
+  invokeTextCompletionsApi as invokeClaude2TextCompletionsApi,
+  invokeMessagesApi as invokeClaude2MessagesApi,
+} from "../models/anthropic_claude/claude_2.js";
+import {
+  invokeModel as invokeClaude3,
+  invokeModelWithResponseStream as invokeClaude3WithResponseStream,
+} from "../models/anthropic_claude/claude_3.js";
+
+const TEXT_PROMPT = "Hello, this is a test prompt";
+
+describe("Invoke Anthropic Claude Instant using the Text Completions API", () => {
+  it("should return a response", async () => {
+    const modelId = FoundationModels.CLAUDE_INSTANT.modelId;
+    const response = await invokeClaudeInstantTextCompletionsApi(
+      TEXT_PROMPT,
+      modelId,
+    );
+    expectToBeANonEmptyString(response);
+  });
+});
+
+describe("Invoke Anthropic Claude Instant using the Messages API", () => {
+  it("should return a response", async () => {
+    const modelId = FoundationModels.CLAUDE_INSTANT.modelId;
+    const response = await invokeClaudeInstantMessagesApi(TEXT_PROMPT, modelId);
+    expectToBeANonEmptyString(response[0]);
```

Comment:
The amok feeling is here too. If `response` is a string instead of an array, this would still pass. e.g. `"a response"[0]` would return `"a"`.

Reply:
As mentioned above, not all invocation methods return a string, some return an array. I'll have a look at how I can make this either clearer or unify across all of them.